xarray/test/test_dataset.py

```diff
@@ -2494,3 +2494,36 @@ def test_setattr_raises(self):
             ds.foo = 2
         with self.assertRaisesRegexp(AttributeError, 'cannot set attr'):
             ds.other = 2
+
+    def test_get_variables_by_attributes(self):
+        ds = Dataset({'temperature_0': (['t'],  [0], dict(standard_name='air_potential_temperature', height='0 m')),
+                      'temperature_10': (['t'],  [0], dict(standard_name='air_potential_temperature', height='10 m')),
+                      'precipitation': (['t'], [0], dict(standard_name='convective_precipitation_flux'))},
+                    coords={'time': (['t'], [0], dict(axis='T'))})
```

Comment:
nit: can you fix the styling here? These lines are pretty long and there are some extra spaces before the `[0]`s.